index.html

```diff
@@ -8,65 +8,32 @@
 
 </head>
 <body>
-<table rules="groups">
-    <tr>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-    </tr>
-    <tr>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-    </tr>
-    <tr>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-    </tr>
-    <tr>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-        <td>
-            <div class="coub"></div>
-        </td>
-    </tr>
-</table>
 
-</table>
+<div class="table">
+    <div class="row">
+        <div class="coub"></div>
```

Comment:
coub -> cell

Reply:
Everyone names it cell but I wan to be uniq :)